go/runfiles/manifest.go

```diff
@@ -138,7 +138,8 @@ func (m *manifest) open(name string) (fs.File, error) {
 			// name refers to an actual file or dir listed in the manifest. The
 			// basename of name may not match the basename of the underlying
 			// file (e.g. in the case of a root symlink), so patch it.
-			f, err := os.Open(r)
+			var f *os.File
```

Comment:
accidental change?

Reply:
I needed this to have `nogo`'s `shadow` analyzer pass on the runfiles lib. Since we don't control the analyzers it is built with, I guess we just have to ensure that it's passing all the ones we know of.